src/content/guides/getting-started.mdx

```diff
@@ -317,6 +324,10 @@ webpack 5.4.0 compiled successfully in 1940 ms
 
 T> Custom parameters can be passed to webpack by adding two dashes between the `npm run build` command and your parameters, e.g. `npm run build -- --color`.
 
+For the live preview, once the build is finished, you can run `npm run start` to serve the `dist` folder.
```

Comment:
```suggestion
For the live preview, once the build is finished, you can run `npm start` to serve the `dist` folder.
```

Reply:
This is not necessary. Do you still want to add `run` to it?